### PR TITLE
Documents

### PR DESCRIPTION
**Pull request by MathiasGagnepain**
- weekly 5
- update security
- technical specifications

### CODE REVIEW

**Review by EnzoGuillouche — APPROVED** [review]
Reviewed all the documents. Merging approved.

File: documents/QA/test_plan.md
Comment (MaximeAlgosup) [inline documents/QA/test_plan.md:1]: Maybe the 3D and the pink background are a bit too much @EnzoGuillouche

File: documents/technical/technical.md
Comment (MaximeAlgosup) [inline documents/technical/technical.md:1]: @Guillaume18100 Is the description of each .c and .h file really necessary? I mean there lot of files and there is no real interest in describing all of them. By the way, all files are not already created and some will maybe disappear

**Review by MaximeAlgosup — APPROVED** [review]
I approve the requests but some things have to be revised

**Pull request merged**